src/extension/options-page/DashboardComponents/Carousel.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from 'react'
+import { useInterval } from 'react-use'
+import { Fade } from '@material-ui/core'
+
+export interface CarouselProps {
+    items: React.ReactElement[]
+    delay?: number
+}
+
+export function Carousel({ items, delay = 1e4 }: CarouselProps) {
```

Comment:
Maybe `FadeInList`?

Reply:
It's not about the list. It's fade items in the list.